respeaker_ros/launch/sample_respeaker.launch

```diff
@@ -13,23 +13,30 @@
   <arg name="language" default="en-US"/>
   <!-- self cancellation -->
   <arg name="self_cancellation" default="true"/>
+  <!-- audio info topic name -->
+  <arg name="audio_info" default="audio_info"/>
 
   <node if="$(arg publish_tf)"
         name="static_transformer" pkg="tf" type="static_transform_publisher"
         args="0 0 0 0 0 0 1 map respeaker_base 100"/>
 
   <node if="$(arg launch_respeaker)"
         name="respeaker_node" pkg="respeaker_ros" type="respeaker_node.py"
-        respawn="true" respawn_delay="10" />
+        respawn="true" respawn_delay="10" >
+    <rosparam subst_value="true" >
+      publish_multichannel: $(arg publish_multichannel)
+    </rosparam>
+  </node>
 
   <node if="$(arg launch_soundplay)"
         name="sound_play" pkg="sound_play" type="soundplay_node.py"/>
 
   <node name="speech_to_text" pkg="respeaker_ros" type="speech_to_text.py"
         output="screen">
-    <remap from="audio" to="$(arg audio)"/>
+    <remap from="audio" to="/speech_audio"/>
```

Comment:
is this change really necessary?

Reply:
Sorry, I mistakenly commit it.
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/350/commits/206aaaadb7e1521fb0cd1db651be5f11f6e7cc3b
I modified it.
I'll rebase if necessary. Or I hope squash merge.